automation_tools/__init__.py

```diff
@@ -3225,3 +3226,12 @@ def setup_alternate_capsule_ports(port_range='9400-14999'):
     # labelling custom port range so that passenger will be allowed to connect
     run('semanage port -a -t websm_port_t -p tcp {0}'
         .format(port_range), warn_only=True)
+
+
+def setup_rhv_ca():
+    """ Setups and trusts the provided CA cert
+    """
+    http_server = os.environ.get('HTTP_SERVER_HOSTNAME')
+    run('wget {0}/pub/rhevm1'.format(http_server))
```

Comment:
i'm a little confused here - does it matter where you download the CA cert? how does `update-ca-trust` command find it?
don't you need to specify some special directory to download the CA cert to?

Reply:
My bad, will correct this